app/models/participant_declaration/npq.rb

```diff
@@ -10,6 +10,7 @@ class ParticipantDeclaration::NPQ < ParticipantDeclaration
     "npq-headship" => "NPQH",
     "npq-executive-leadership" => "NPQEL",
     "npq-early-years-leadership" => "NPQEYL",
+    "npq-leading-primary-mathematics" => "NPQLPM",
```

Comment:
We can keep this, but as mentioned was waiting for NPQ team, but yea let's keep it :)